app/controllers/api/v0/business_days_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V0::BusinessDaysController < ApiController
+  def show
```

Comment:
yes public! is that problematic? I also didn't add to swagger docs because I don't imagine external parties will care about it much